### PR TITLE
Initial drop of the installation docs

### PR DESCRIPTION
**Pull request by chilanti**

### CODE REVIEW

File: docs/getting-started/installation.md [hunk 1]
```diff
@@ -1,10 +1,80 @@
...
-<!-- This will be documentation for installing the appsody CLI 
-This will cover installation for various OS:
-- macOS
-- windows
-- linux
-(This should also include prereqs for using appsody and maybe link to completions) -->
\ No newline at end of file
+Here we cover the steps to get Appsody installed on your system. 
+
+Currently, Appsody is provided on these four OS platforms:
```
Comment (helenmasters) [inline docs/getting-started/installation.md]: If these are pre-built binary releases how about - 
"Appsody provides pre-built binary releases for four platforms:"

File: docs/getting-started/installation.md [hunk 2]
```diff
@@ -1,10 +1,80 @@
...
+1) macOS (we developed and tested primarily on Mojave)
+2) Ubuntu (We tested primarily on 18.04.2)
+3) RedHat Linux (RHEL - we tested on 7.4)
```
Comment (helenmasters) [inline docs/getting-started/installation.md]: "RedHat Linux (we tested on RHEL 7.4)" or "RedHat Enterprise Linux (we tested on 7.4)"

File: docs/getting-started/installation.md [hunk 3]
```diff
@@ -1,10 +1,80 @@
...
+
+Follow these steps: 
+1) Download the **Appsody binaries for Windows** from  [Appsody releases page](https://github.com/appsody/appsody/releases). The file is named `appsody-v.r.m-windows.tar`, where `v.r.m` indicates the release tag.
+2) Move the file to a directory where you want to keep the Appsody binaries (say, an `appsody` folder under your home folder.)
+3) Untar the file there: `tar -xvf appsody-v.r.m-windows.tar`.
+4) Run the setup command `appsody-setup.bat`.
+
+You can now use Appsody.
+
+If you want to remove Appsody, just delete the directory where you extracted the binaries. You may also want to remove the `.appsody` directory that gets created under your home folder. And finally, you may want to remove the path entry that was created for you by the setup command (instructions on editing your path env var can be found [here](https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/)).
```
[the remark this answers is not in the document]
Comment (chilanti) [reply docs/getting-started/installation.md]: On Windows, the path environment var is not necessarily spelled all caps. It's most often spelled Path.

**Comment by chilanti** [comment]
All these comments were addressed in a separate pull request (https://github.com/appsody/docs/pull/10) - please close this one.

**Pull request closed without merging**